src/api-auth-form/submit-button.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var $ = require('jquery');
+
+var decoratorHelper = require('./decorator-helper');
+
+var DEFAULT_SUBMIT_BUTTON_CLASS = 'sso-submit';
+
+function SubmitButton (options) {
+  this.options = options;
+  if (!this.options.submitButton) {
+    this.options.submitButton = {};
+  }
+  this.$button = this.constructButton();
+}
+
+SubmitButton.prototype.constructButton = function () {
```

Comment:
We should stick to the same standard defined in the field.js initialization. Either this constructor is init or constructor. I personally prefer init in all cases.